configs/diem_improvement_proposals.json

```diff
@@ -0,0 +1,28 @@
+{
+  "index_name": "diem_improvement_proposals",
+  "start_urls": [
+    "https://dip.diem.com/",
+    "https://dip.diem.com/overview",
+    "https://dip.diem.com/all-dips"
+  ],
+  "sitemap_urls": [
+    "https://dip.diem.com/sitemap.xml"
+  ],
+  "sitemap_alternate_links": true,
+  "stop_urls": [],
+  "selectors": {
+    "lvl0": "article h1",
+    "lvl1": "article h2",
+    "lvl2": "article h3",
+    "lvl3": "article h4",
+    "lvl4": "article h5",
+    "text": "article p, article li"
+  },
+  "selectors_exclude": [
+    ".hash-link"
+  ],
+  "conversation_id": [
+    "1129563849"
+  ],
+  "nb_hits": 187
+}
```

Comment:
I use the same values here as another config - https://github.com/algolia/docsearch-configs/blob/master/configs/libra-lips.json

I hope that is ok?

Reply:
All good, I updated it to make sure it match your website structure.